src/inference/include/ie/ie_parallel.hpp

```diff
@@ -129,6 +133,69 @@ inline void parallel_set_num_threads(int) {
 #endif
 
 namespace InferenceEngine {
+#if ((IE_THREAD == IE_THREAD_TBB) || (IE_THREAD == IE_THREAD_TBB_AUTO))
```

Comment:
Should it be a part of this PR?

Reply:
good point, will move to some internal new (?) header (common) instead